lib/base-chips/field_type.hashed-text.js

```diff
@@ -0,0 +1,44 @@
+var Sealious = require('../main.js');
+var Promise = require("bluebird");
+var crypto = require("crypto");
+
+var field_type_hashed_text = new Sealious.ChipTypes.FieldType({
+	name: "hashed-text",
+	extends: "text",
+	get_description: function(context, params){
+		return "Hash text with chosen algorithm (default md5)";
+	},
+	is_proper_value: function(accept, reject, context, params, new_value){
+		var pattern_array = [];
+		if (params instanceof Object){
+			if (params.digit)
+				pattern_array.push(new RegExp("(.*[0-9]+.*){" + params.capitals + "}"));
+			if (params.capitals)
+				pattern_array.push(new RegExp("(.*[A-Z]+.*){" + params.capitals + "}"));
```

Comment:
If someone writes `{name: "password", type: "hashed-text", {digits: 2, capitals: 5} etc.}` then it's not ambiguous.

I can change the names to `digits_required` and `capitals_required`, "n" is redundant. :)